js_modules/dagit/packages/ui/src/components/ConfigEditorWithSchema.tsx

```diff
@@ -18,8 +18,9 @@ interface Props {
 }
 
 // Force code editor hints to appear above the dialog modal
-const CodeMirrorShimStyle = createGlobalStyle`
-  .CodeMirror-hints {
+export const CodeMirrorInDialogStyle = createGlobalStyle`
+  .CodeMirror-hints,
+  .CodeMirror-hints.dagit {
```

Comment:
Did this just need a specificity bump? If so, can it be `&.CodeMirror-hints` or something? It might be kind of nice to avoid needing to use the `dagit` class here specifically, since it's a `dagster-io/ui` component and we won't always have that classname on its use cases.